panic due to nil map when decoding Conway redeemers

Seen in `preview`

```
panic: assignment to entry in nil map

goroutine 1406 [running]:
github.com/blinklabs-io/gouroboros/ledger.(*ConwayRedeemers).UnmarshalCBOR(0xc0012d5740, {0xc00139e069, 0x12, 0xd97})
        /home/agaffney/go/pkg/mod/github.com/blinklabs-io/gouroboros@v0.91.0/ledger/conway.go:153 +0x1cd
```

--- a/ledger/conway.go
+++ b/ledger/conway.go
@@ -141,6 +141,7 @@ func (r *ConwayRedeemers) UnmarshalCBOR(cborData []byte) error {
 	var tmpRedeemers []AlonzoRedeemer
 	if _, err := cbor.Decode(cborData, &tmpRedeemers); err == nil {
 		// Copy data from legacy redeemer type
+		r.Redeemers = make(map[ConwayRedeemerKey]ConwayRedeemerValue)
 		for _, redeemer := range tmpRedeemers {
 			tmpKey := ConwayRedeemerKey{
 				Tag:   redeemer.Tag,

